TheMovieDb/Reviews/View/ReviewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ReviewCell.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit
+
+final class ReviewCell: UICollectionViewCell {
+    
+    // MARK: - Properties
+    static let reuseIdentifier =  String(describing: ReviewCell.self)
+
+    public var review: Review? {
```

Comment:
by default all properties are internal, why you need public here ?

Reply:
Currently I'm using a view model, because was a little logic in function configure.
```suggestion
        public var viewModel: ReviewViewModel? {
        didSet {
            configure()
        }
    }
```